src/packages/utils/src/things/json-rpc/json-rpc-quantity.ts

```diff
@@ -2,8 +2,10 @@ import { bufferToBigInt } from "../../utils/buffer-to-bigint";
 import { BaseJsonRpcType } from "./json-rpc-base-types";
 import { JsonRpcInputArg } from "./input-parsers";
 import { BUFFER_EMPTY, BUFFER_ZERO } from "../../utils/constants";
+import { bigIntToBuffer } from "../../utils/bigint-to-buffer";
 
 export class Quantity extends BaseJsonRpcType {
+  public static Null = Quantity.from(null, true);
```

Comment:
It doesn't look like this is used. Am I wrong?

Reply:
It's also effectively the same as `Quantity.Empty`